packages/kbn-dev-utils/package.json

```diff
@@ -1,26 +1,25 @@
 {
   "name": "@kbn/dev-utils",
   "main": "./target/index.js",
-  "types": "./index.d.ts",
   "version": "1.0.0",
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build": "babel src --out-dir target --copy-files",
-    "kbn:bootstrap": "yarn build --quiet",
+    "build": "tsc",
+    "kbn:bootstrap": "yarn build",
     "kbn:watch": "yarn build --watch"
   },
   "dependencies": {
     "chalk": "^2.4.1",
     "execa": "^1.0.0",
+    "exit-hook": "^2.2.0",
     "moment": "^2.20.1",
     "rxjs": "^6.2.1",
     "tree-kill": "^1.2.0",
     "tslib": "^1.9.3"
   },
   "devDependencies": {
-    "@babel/cli": "7.4.4",
-    "@kbn/babel-preset": "1.0.0",
+    "typescript": "3.5.3",
```

Comment:
Same question here, this dep is in both spots.

Reply:
Same answer :)